wrapping/itkTileMergeImageFilter.wrap

```diff
@@ -1,10 +1,13 @@
 itk_wrap_class("itk::TileMergeImageFilter" POINTER)
   foreach(d ${ITK_WRAP_IMAGE_DIMS})
     foreach(t ${WRAP_ITK_SCALAR})
-      itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}")
+      itk_wrap_template("${ITKM_I${t}${d}}" "${ITKT_I${t}${d}}, double")
```

Comment:
- Is this already the default wrapping for this class? [`itkTileMergeImageFilter.h`](https://github.com/InsightSoftwareConsortium/ITKMontage/blob/4ed5b6051b6162c464a190a384553191d312eecb/include/itkTileMergeImageFilter.h#L47) defines default template arguments:
```cpp
template <typename TImageType,
          typename TPixelAccumulateType = typename NumericTraits<typename TImageType::PixelType>::AccumulateType,
          typename TInterpolator = LinearInterpolateImageFunction<TImageType, float>>
class ITK_TEMPLATE_EXPORT TileMergeImageFilter
```
And in [`itkNumericTraits.h`](https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/include/itkNumericTraits.h#L74) `AccumulateType` is already `double` regardless of `T`:
```cpp
using AccumulateType = double;
```

Reply:
That is the case for unspecialized, unknown types. All the usual types are specialized and have a reasonable choice, e.g. `bool`'s accumulation type is [char](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.3rc02/Modules/Core/Common/include/itkNumericTraits.h#L264).

Reply:
Ah makes sense, thanks!